_sass/_variables.scss

```diff
@@ -28,3 +28,6 @@ $code-font-size: 0.8em;
 
 $sidebar-width: 300px;
 $sidebar-padding-sides: 15px;
+
+$average-ai-content-card-height: 400px;
+$width-when-desktop-gets-crowded: 300px;
```

Comment:
A suggestion with the naming here - instead of just adding these two variables, you can always define new ones that are composed of previous ones! Ex:

```scss
$width-when-desktop-gets-crowded: 300px;
$desktop-width-xl: $desktop-width + $width-when-desktop-gets-crowded;
```

And then you can use that class where you want!

Reply:
would it be better to just make `$desktop-width-xl: $desktop-width + 300px;` instead of doing what you wrote. Since $width-when-desktop-gets-crowded kind of feels like a weird variable that is being defined only for the use of one other variable.

Reply:
Oh yes, that is totally fine - great idea!